code_annotations/report_templates/annotation_list.tpl

```diff
@@ -10,6 +10,12 @@ Annotations found in {{ report|length }} files.
     {% for annotation in report[filename] %}
         {% if annotation.report_group_id %}
             {% if loop.changed(annotation.report_group_id) %}
+
+                {% if annotation.annotation_token == '.. toggle_name:' %}
```

Comment:
It's been a while since I've been in here, but I'm not sure this line is necessary? If so, we should not be hard coding the annotation token, since any given configuration may not have this token in it. Is the goal to have an anchor for the first annotation in each group?

Reply:
Ah, good point. I was only thinking of my needs here. Yes, that is the goal. I see what you are saying about it not being necessary if we use the `loop.changed` part.  I suppose I wanted to the anchor to be on something recognizable, in this case, the name of the toggle, because I am trying to have bidirectional links between this report and another.

Reply:
@bmedx I changed it to be more generic. Can you take another look when you get a chance?